<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_users_dialog</class>
 <widget class="QDialog" name="view_users_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>410</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose User :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>44</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User's name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>71</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current balance:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="transTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>591</width>
     <height>251</height>
    </rect>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <zorder>comboBox</zorder>
  <zorder>label</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_2</zorder>
  <zorder>label_20</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
